main.tf

```
@@ -137,7 +137,7 @@ module "dns-primaries" {
   install_id = local.install_id
   prefix     = var.prefix
 
-  project   = var.project
+  project   = local.dns_project
```

per the other comment for the locals block

```suggestion
  project   = local.rendered_dns_project
```